staging/src/k8s.io/component-base/featuregate/feature_gate.go

```diff
@@ -309,7 +318,27 @@ func (f *featureGate) Validate() []error {
 		return []error{fmt.Errorf("cannot cast enabledRaw to map[string]bool")}
 	}
 	enabled := map[Feature]bool{}
-	return f.unsafeSetFromMap(enabled, m, f.EmulationVersion())
+	errs := f.unsafeSetFromMap(enabled, m, f.EmulationVersion())
```

Comment:
if errs is non-empty, maybe just return? not sure it's helpful to add more errors about relationships between gates when the gates that are set aren't even what they asked for